### ISSUE TITLE
Add Installation Instructions to README about building from .tar.gz

### ISSUE THREAD

**Issue by hussein-esmail7** [clangen/musikcube#486]
I was having issues building the latest version of musikcube available on the AUR ([0.96.10-1](https://aur.archlinux.org/packages/musikcube/)), so I had to install it from an older version (I knew [0.96.8](https://github.com/clangen/musikcube/releases/tag/0.96.8) worked on my other machine so I used that) by downloading the .tar.gz file in the [Releases](https://github.com/clangen/musikcube/releases/) tab on this Github repo. 

The installation of it was not mentioned specifically anywhere and the generic .tar.gz installation instructions were not identical so I was wondering if specific instructions can be added to the [Installation Guide](https://github.com/clangen/musikcube/wiki/installing) page.

Here's what worked for me:

`tar -xzf musikcube-0.96.8.tar.gz`
`cd musikcube-0.96.8/`
`cmake .`
`make .`
`sudo make install .`

On other forums, the instructions are similar but they don't have the `cmake .` step.

Information about my machine:
`Kernel: 5.15.11-arch2-1`
`Shell: bash 5.1.12`

**Comment by clangen** [comment]
Thanks, I can add these instructions to the wiki, but can you tell me about the build issues you experienced with 0.96.10? What failed? I have an Arch installation and it seems fine.

**Comment by hussein-esmail7** [comment]
@clangen [Here is a Gist of all the output from installing it via yay](https://gist.github.com/hussein-esmail7/7c6970f886cb0d32cfab88e76cc55cbb)

**Comment by clangen** [comment]
Build instructions on the wiki have been updated as requested.

I took a look at your build logs, and it seems `yay` is trying to build `0.96.7` -- you can see the version number in the output:

```
/home/hussein/.cache/yay/musikcube/src/musikcube-0.96.7/src/plugins/httpdatastream/LruDiskCache.cpp:305:21: error: base operand of '->' is not a pointer
  305 |         return entry->id == id;
      |                     ^~
```

This particular issue was fixed as part of `0.96.10`.

```
> yay -Ss musikcube
aur/musikcube-git 0.96.7.r5.g00cf3eb5-1 (+2 0.00) 
    a terminal-based cross-platform music player, audio engine, metadata indexer, and server
aur/musikcube 0.96.10-1 (+17 1.21) 
    A terminal-based cross-platform music player, audio engine, metadata indexer, and server
```

I think perhaps you tried to install `musikcube-git` instead of `musikcube` -- could these be the case? Note that I don't maintain either of these ports, and it looks like `musikcube-git` is a bit out of date.

**Comment by hussein-esmail7** [comment]
I don't believe I was installing musikcube, not musikcube-git, as you can see on Line 16 of the [Gist](https://gist.github.com/hussein-esmail7/7c6970f886cb0d32cfab88e76cc55cbb). Either way, I can see on the [musikcube AUR package page](https://aur.archlinux.org/packages/musikcube/) that the version has been updated today so that should no longer be an issue for other people. 

Also thanks for including the `cmake -G "Unix Makefiles" .` in the [build instructions page](https://github.com/clangen/musikcube/wiki/building) since when I just typed `cmake .` no audio drivers were detected, so I actually had to redo my install. Just a note for any future readers here if you also have that issue.

Closing this issue now since the original ask is now resolved.
